modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase

```diff
@@ -149,12 +149,58 @@ definition {
 	}
 
 	@description = "Verify that the users are able to identify the past stages that the workflow instance has already passed by"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanIdentifyPastStagesOnWorkflow {
 
-		// TODO LRQA-72393 CanIdentifyPastStagesOnWorkflow pending implementation
+		property portal.acceptance = "true";
+		property test.name.skip.portal.instance = "WorkflowMetricsAllItemsList#TooltipDisplaysBulletListWhenMultipleItems";
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Process Builder");
+
+		Workflow.uploadDefinition(
+			workflowDefinitionFile = "multi-task-process-workflow-definition.xml",
+			workflowDefinitionTitle = "Multi Task Process");
+
+		Navigator.gotoNavItem(navItem = "Configuration");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Multi Task Process",
+			workflowResourceValue = "Blogs Entry");
+
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry",
+			entryTitle = "Blogs Entry Title");
+
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
+
+		Navigator.gotoNavTab(navTab = "Assigned to My Roles");
+
+		LexiconEntry.gotoEllipsisMenuItem(menuItem = "Assign to Me");
+
+		Workflow.confirmWorkflowAction();
+
+		WorkflowMetrics.goToWorkflowAllItems(workflowProcessName = "Multi Task Process");
+		
+		WorkflowMetrics.goToBulkTransitionSelectTasks();
+
+		WorkflowMetrics.selectAllTransitionTasks();
+
+		WorkflowMetrics.chooseTransitionPerStep(
+			taskName = "Task1",
+			transitionOption = "Join");
+
+		WorkflowMetricsAllItemsList.viewDetailsModal();
+
+		WorkflowMetricsAllItemsList.gotoTrackWorkflowlink();
+
+		WorkflowMetrics.viewNodeCircleIcon(currentNode = "task2");
+
+		WorkflowMetrics.viewNodeCircleIcon(currentNode = "task3");
 
+        WorkflowMetrics.notViewNodeCircleIcon(currentNode = "Fork , Task1");
```

Comment:
Create a macro to assert that the Fork node and Task1 (or any node in that condition) have the done icon present, not just that dont have the green circle icon.